### PR TITLE
Add support for custom database user and name.

### LINKED ISSUE #289: Zulip Container tries to connect to postgres as user 'zulip' regardless of if postgres user is different in docker-compose.yml
## Scenario:

In `docker-compose.yml` you set:

DB_USER to something other than 'zulip'

## Expcected Result:

Zulip should try to connect to the postgres database as whatever you have set DB_USER to.

## Actual Oucome

Zulip container exits straight away as it can't connect to database as the user 'zulip' doesn't have permissions.

### PR DESCRIPTION
**Pull request by adambirds**
Fixes #289.

### CODE REVIEW

**Comment by timabbott** [comment]
This looks correct to me.  I think it should be testable with `docker up` on one's laptop starting from a clean checkout, after changing the database name settings, and seeing if you get a working server.

**Comment by andersk** [comment]
This didn’t work; see [discussion](https://chat.zulip.org/#narrow/stream/2-general/topic/Change.20DB.20user.20and.20name/near/1188913).

TL;DR: I don’t think we should be expected to support customizing the database name for *our own* database. The custom database name feature is for connecting to a separate centrally or externally managed PostgreSQL instance where the administrator decides what the name is.
